pom.xml

```diff
@@ -635,6 +638,7 @@
                 <configuration>
                    <!-- testsToExclude come from the profile-->
                    <excludedGroups>${testsToExclude}</excludedGroups>
+                    <argLine>-Duser.timezone=${project.timezone} -Dfile.encoding=${project.build.sourceEncoding} -Duser.language=${project.language} -Duser.region=${project.region}</argLine>
```

Comment:
Just a heads up that we pass some command line arguments in the `.travis.yml` file in the root of the repo. Tests are passing just find at https://travis-ci.org/IQSS/dataverse without any timezone, language, or region specified but I just thought I'd mention this.

Reply:
Yes of course this works... :wink: 
Travis CI by [default uses locale en_US.UTF-8](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) and sets TZ to UTC according to some stuff on StackOverflow and Github issues.

Reply:
Ok, did you see my comment above on `dev-environment.rst `?

Reply:
Yepp - was AFK for a break... :wink: Had some :cn: dinner.